.travis.yml

```diff
@@ -6,11 +6,33 @@ cache:
     - $HOME/.cache/pip
     - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
 
+before_install:
+  - |
+    set -e
+    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+        arch="Linux"
+    elif [ "$TRAVIS_OS_NAME" == "osx" ]; then
+        arch="MacOSX"
+    else
+        echo "Unknown arch $TRAVIS_OS_NAME"
+        exit 1
+    fi
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-${arch}-x86_64.sh -O miniconda.sh
+    chmod +x miniconda.sh
+    ./miniconda.sh -b -p ~/mc
+    source ~/mc/etc/profile.d/conda.sh
+    conda update conda --yes
```

Comment:
This step may take some time. I think it's OK to not do it as we are going to use the latest distribution available.

Reply:
You are talking about line 24 here correct?
`conda update conda --yes`

Reply:
Yes, just this one line.